bin/all_sky_search/pycbc_coinc_statmap

```diff
@@ -307,21 +313,21 @@ orig_fore_stat = fore_stat
 if args.max_hierarchical_removal != 0:
     # If user wants to remove against inclusive background.
     if args.hierarchical_removal_against == 'inclusive':
-        ifar_foreground = ifar
+        ifar_foreground_yrs = conv.sec_to_year(ifar)
```

Comment:
being very picky I prefer 'yr' as unit - once we resolve other stuff I can just do a find-replace, so no action immediately